apps/web/src/utils/usernames.ts

```diff
@@ -42,6 +42,8 @@ export enum UsernameTextRecordKeys {
   Discord = 'com.discord',
 
   Avatar = 'avatar',
+
+  Frame = 'frame',
```

Comment:
@kirkas thoughts on this value?

Reply:
maybe plural if we save multiple frames URLs under this textRecord?